### PR TITLE
Add BarDrobe and Bar Backroom into Saltern

### PR DESCRIPTION
**Pull request by BulletGrade**
## About the PR 
This PR adds BarDrobe into the base game and a layout change which adds bar backroom into Saltern.
The backroom contains a BarDrobe and a booze storage (which was moved from the bar)
The BarDrobe contains:
- 2 tophats
- 2 beer goggles
- 2 service headsets
- 2 bartender jumpsuit
- 2 bartender jumpskirt
- 2 purple bartender's jumpskirt
- 2 black shoes
- 1 kevlar vest

Sprites were taken from /tg/station
**Screenshots**

**Changelog**

:cl: Hardly
- add: Bar backroom in Saltern
- add: BarDrobe

### CODE REVIEW

**Review by Zumorica — CHANGES_REQUESTED** [review]
It seems you didn't fix the roundstart atmosphere, please do that as shown [here](https://hackmd.io/1CmC4RpnStSAIOKWWa1fyw#How-to-fix-roundstart-atmos-after-making-changes-to-the-map)

**Comment by Zumorica** [comment]
Changelog seems broken, other than that looks good

**Comment by BulletGrade** [comment]
That should fix it

File: Resources/Textures/Constructible/Power/VendingMachines/bardrobe.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            ]
+        },
+        {
+            "name": "broken",
+            "directions": 1,
+            "delays": [
+                [
+                    1.0
+                ]
+            ]
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Power/VendingMachines/bardrobe.rsi/meta.json:33]: Don't need as they're defaults

File: Resources/Textures/Constructible/Power/VendingMachines/bardrobe.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/tgstation/tgstation/blob/master/icons/obj/vending.dmi",
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Power/VendingMachines/bardrobe.rsi/meta.json:8]: Please use specific commit.

File: Resources/Maps/saltern.yml [hunk 3]
```diff
@@ -47709,4 +47668,62 @@ entities:
...
+- uid: 4884
+  type: AirlockMaint
+  components:
+  - pos: -14.5,-9.5
+    parent: 853
+    type: Transform
+  - containers:
+      board: !type:Container
+        ents: []
+    type: ContainerContainer
```
Comment (metalgearsloth) [inline Resources/Maps/saltern.yml:47702]: @Zumorica these aren't supposed to be here right?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Maps/saltern.yml (hunk 3 above)
Comment (Zumorica) [reply Resources/Maps/saltern.yml:47702]: It's fine as there's not an entity inside it

**Comment by BulletGrade** [comment]
The meta file should be good now, if there's anything that needs to be changed let me know.
BTW. I have added Black Shoes into the Bardrobe.

**Review by SweptWasTaken — APPROVED** [review]

**Pull request merged**